src/org/daisy/dotify/formatter/impl/writer/PEFMediaWriter.java

```diff
@@ -72,7 +73,7 @@ public void open(OutputStream os) throws PagedMediaWriterException {
         open(os, null);
     }
 
-    private void open(OutputStream os, List<MetaDataItem> data) throws PagedMediaWriterException {
+    public void open(OutputStream os, List<MetaDataItem> data) throws PagedMediaWriterException {
```

Comment:
Could not revert this change as this is the way we add the namespaces to the output. I did not change this function because I wanted to test it. I changed it because we use it in our implementation and the test verifies that it works.

Reply:
@kalaspuffar I guess I didn't notice this change before. I don't understand. How exactly are you using this class?

Reply:
My bad made function private and removed test.